src/api/schema/metrics/host.rs

```diff
@@ -191,7 +191,7 @@ impl NetworkMetrics {
 
     /// Total transmission packets dropped (Linux/Windows only)
     async fn transmit_packets_drop_total(&self) -> Option<f64> {
-        if cfg!(any(target_os = "linux", target_os = "windows")) {
+        if cfg!(any(target_os = "linux", windows)) {
```

Comment:
```suggestion
        if cfg!(any(linux, windows)) {
```

This would be the case you mentioned looking for, right?

Reply:
No, I don't believe there is a `cfg(linux)`, only `cfg(unix)`, which means `cfg(target_family = "unix")` and includes Linux, MacOS, BSD, etc.

Reply:
Ah, _booo_. So much for consistency 😄